vm-virtio/src/iommu.rs

```diff
@@ -383,14 +396,22 @@ impl Request {
                     return Err(Error::InvalidDetachRequest);
                 }
 
-                let req: VirtioIommuReqAttach = mem
+                let req: VirtioIommuReqDetach = mem
```

Comment:
hah, we've been lucky :-)

Reply:
Yes, that's because both structure are identical.